### PR TITLE
cleanup urdfdom compatibility

### PR DESCRIPTION
**Pull request by rhaschke**
... now relying on compatibility layer of urdf package: https://github.com/ros/robot_model/pull/160

### CODE REVIEW

**Comment by v4hn** [comment]
Should we hold off this change until the robot_model update is synced?
Otherwise would this require changes to the buildfarm again so that tests of other requests still succeed?

**Comment by davetcoleman** [comment]
+1 to holding off until sync

**Comment by v4hn** [comment]
This is overdue. Merging.

File: include/srdfdom/model.h [hunk 1]
```diff
@@ -41,8 +41,7 @@
 #include <string>
 #include <vector>
 #include <utility>
-#include <urdf_model/model.h>
-#include <srdfdom/urdfdom_compatibility.h>
+#include <urdf/model.h> // TODO: replace with urdf_model/types.h in Lunar
```
Comment (130s) [inline include/srdfdom/model.h:44]: > // TODO: replace with urdf_model/types.h in Lunar

Does this still hold?

Comment (davetcoleman) [reply include/srdfdom/model.h:44]: I tried replacing it on a Ubuntu Zesty system and got an error:

> /root/ros/ws_catkin/src/srdfdom/include/srdfdom/model.h:66:28: error: 'ModelInterface' in namespace 'urdf' does not name a type

Which I have not looked into

**Pull request merged**